source/bin/tvm.rs

```diff
@@ -114,14 +115,24 @@ pub fn process_options(arguments: Vec<String>) {
                     },
 
                     _ => {
-                        println!("Invalid option “{}”.\n", argument);
+                        println!("Invalid option \"{}\".\n", argument);
                         println!("{}", usage());
                         exit(ExitCode::InvalidOption);
                     }
                 },
 
-            Some(_) =>
-                input = argument,
+            Some(_) => {
+                if input == None
+                {
+                    input = Some(argument);
```

Comment:
Quote from `python -c 'import this'`:

> Errors should never pass silently.
> Unless explicitly silenced.

Try to go through your code what happens with the following input:

`cmd input1.php input2.php`